requirements.txt.in

```diff
@@ -3,7 +3,7 @@ azure-storage >= 0.36.0
 boto3 >= 1.6.0
 botocore >= 1.10.16  # 1.9.x does not support AWS secretsmanager support
 cloud-blobstore >= 2.1.5
-connexion == 1.1.15 # pinned by akislyuk due to upstream breaking changes in auth middleware in connexion 1.2
+connexion >= 1.1.15
```

Comment:
Nice, did you notice any differences with the latest connexion?

Reply:
nothing significant other than the addition of tokeninfo_func. I'm not sure if this as there before but `app.add_error_handler()` was a nice feature.